packages/okta-react/package.json

```diff
@@ -61,9 +64,18 @@
     "jest": "^22.4.2",
     "polished": "^1.7.0",
     "protractor": "^5.1.2",
-    "react-dom": "^15.6.1",
+    "react": ">=16",
+    "react-dom": ">=16",
```

Comment:
add react-dom and react dev-deps as version 16 so tests are compatible with the enzyme adapter. Peer dependencies are still the same.